templates/level-page.html

```diff
@@ -24,12 +24,18 @@
               else if not arrow.classList.contains('rotate-180')
                   add .rotate-180 to arrow
               end">
-            {% for dropdown_level in customizations['available_levels'] %}
+            {% for dropdown_level in range(1, max_level) %}
               <button id="level_button" class="flex-row flex justify-between items-center gap-2 px-2 py-2 border-b border-dashed border-blue-500
-              {% if dropdown_level == level %}bg-white{% else %}hover:bg-blue-100 bg-blue-200{% endif %}"
-              {% if dropdown_level == level %}disabled{% endif %}
+              {% if dropdown_level == level %}bg-white{% elif dropdown_level not in customizations['available_levels'] %}bg-gray-400 tooltip{% else %}hover:bg-blue-100 bg-blue-200c{% endif %}"
+              {% if dropdown_level == level or dropdown_level not in customizations['available_levels']%}disabled{% endif %}
               style="width: 100%;"
               _="on click hedyApp.goToLevel({{dropdown_level}})">
+              <div class='{% if dropdown_level in customizations['available_levels'] %}hidden{% endif %}
+              text-center opacity-0 hover:opacity-100 duration-300 absolute inset-0 z-10 items-top'>
+                <span class='tooltiptext text-xs font-medium absolute rounded shadow-lg' style="margin-left: -50px; bottom: 130%; color:black; background:lightsteelblue;">
+                  {{ _('tooltip_level_locked') }}
+                </span>
+              </div>
```

Comment:
This is definitly the proper way of working with Hyperscript. Amazing! 

Do add some padding to the tooltip and center it as well.

Reply:
Is it better like this?

Reply:
See this:

![image](https://github.com/hedyorg/hedy/assets/20051470/ad0134ef-ba0a-48d5-8339-9a5d4f30b719)

Also, why is it squeezed within the dropdown? Make it free ;))

Reply:
Ah good catch, thanks. I've been trying, can you help me out on how to make it free?

Reply:
Alright, will text you on discord